doc/CONTRIBUTING.md

```diff
@@ -127,10 +122,9 @@ and exposed an API to the front-end for file access and all UI needs.
 
 Over time, Microsoft added support to VS Code to run it on the web. They have made
 the front-end open source, but not the server. As such, code-server v2 (and later) uses
-the VS Code front-end and implements the server. You can find this in
-[./ci/dev/vscode.patch](../ci/dev/vscode.patch) under the path `src/vs/server`.
+the VS Code front-end and implements the server. We use a git subtree to be able to pull in VS Code's front-end. This code lives under [./lib/vscode](./lib/vscode).
 
-Other notable changes in our patch include:
+Some noteworthy changes in our codebase:
 
 - Adding our build file, which includes our code and VS Code's web code
 - Allowing multiple extension directories (both user and built-in)
```

Comment:
Looking for feedback on the wording here

Reply:
Maybe "Some noteworthy changes in our version of VS Code"? 

I keep wanting to say "fork of VS Code" but I'm not sure if that would be confusing since we have it as a subtree. 

For example "changes in our fork of VS Code" or "we do this by using a git subtree to fork and modify VS Code".

Reply:
Updated!